bindings/python/examples/wallet/12-prepare_output.py

```diff
@@ -17,17 +17,9 @@
 wallet.set_stronghold_password(os.environ["STRONGHOLD_PASSWORD"])
 
 # using prepare_output
-output = account.prepare_output(
-    {
-        "amount": "1000000",
-        "recipientAddress": "rms1qprutadk4uc9rp6h7wh7sech42sl0z40ztpgyygr5tf0cn5jrqshgm8y43d",
-        "unlocks":
-            {
-                "expirationUnixTime": 1676570528,
-            },
-    }
-)
-print(f"Output: {json.dumps(output, indent=4)}")
+output = account.prepare_output(OutputParams(
+    "rms1qprutadk4uc9rp6h7wh7sech42sl0z40ztpgyygr5tf0cn5jrqshgm8y43d", "1000000", unlocks=Unlocks(expirationUnixTime=1676570528)))
+print(f"Output: {json.dumps(output.as_dict(), indent=4)}")
```

Comment:
json dumps still necessary?

Reply:
To have the output with the other languages aligned yes